src/views/server.tsx

```diff
@@ -38,16 +41,21 @@ export const handleRender = (
     const finalState = store.getState()
 
     // Send the rendered page back to the client
-    return renderFullPage(html, finalState)
+    return renderFullPage(html, finalState, staticPath, publicPath)
 }
 
-export const renderFullPage = (html: any, preloadedState: any): any => {
+export const renderFullPage = (
+    html: any,
+    preloadedState: any,
+    staticPath?: string,
+    publicPath?: string
+): any => {
     return `
     <!doctype html>
     <html>
       <head>
         <title>Expresso</title>
-        <link rel="shortcut icon" type="image/jpg" href="expresso/favicon.ico"/>
+        <link rel="shortcut icon" type="image/jpg" href="favicon.ico"/>
```

Comment:
Should come from static folder either ?

Reply:
Yes. I miss it. I should add `staticPath` before favicon.